src/masonite/drivers/mail/BaseMailDriver.py

```diff
@@ -40,6 +43,42 @@ def mail_from_header(self):
     def mail_to_header(self):
         return ','.join(self.to_addresses)
 
+    def text(self, content):
+        """Set the text content of the email.
+
+        Arguments:
+            content {string} -- The email text content.
+
+        Returns:
+            self
+        """
+        self.text_content = content
+        return self
+
+    def html(self, content):
+        """Set the html content of the email.
+
+        Arguments:
+            content {string} -- The email html content.
+
+        Returns:
+            self
+        """
+        self.html_content = content
```

Comment:
self.text_content and self.html_content are repetitive. the point of setting one or the other is so later in the building process you can either set the html or plain text mime type.

Reply:
The 2 attributes are not the same thing at all.
The html content has rich html and markup and may even have different text, whereas the text content is a minimal text version including specific line quoting and chars which render differently.
Example 
```python
mail.html("<div>Hello&nbsp;<strong>User</strong></div>").text("Hello User\n message...").send()
```

Reply:
Oh ok. Right you want to use both. Gotcha